docs/Authentication.md

```diff
@@ -27,7 +27,7 @@ For the F5 experience and test environments, an in-process identity provider is
 To enable the development identity provider, add the following code to your app configuration:
 
 ```csharp
-builder.AddDevelopmentAuthEnvironment("path/to/testAuthEnvironment.json");
+builder.AddDevelopmentAuthEnvironmentIfConfigured(configurationRoot);
 ```
 
 The code above, along with the `TestAuthEnvironment.json` file will configure and start the identity provider for use by the FHIR API.
```

Comment:
>The code above, along with the `TestAuthEnvironment.json [](start = 0, length = 56)

nit: I would add few words, something like : `configurationRoot` object should had key "TestAuthEnvironment:FilePath" pointing to `TestAuthEnvironment.json` location. In past it was direct string, so sentence was ok, but now it feels disconnected.

Reply:
fixed, please check.